custom-namespaces.yaml

```diff
@@ -229,3 +229,13 @@ newtab:
   owners:
     - mmccorquodale@mozilla.com
   spoke: looker-spoke-private
+awesome_bar:
+  glean_app: false
+  owners:
+    - tbrooks@mozilla.com
+  pretty_name: Awesome Bar
+  views:
+    urlbar_clients_daily:
+      type: ping_view
```

Comment:
just curious, i see that the type is `ping_view`, what does that mean?

Reply:
I'm not familiar with this, but looking at https://github.com/mozilla/lookml-generator/tree/main/generator/views this seems to be the right parser for this table. Though this wasn't obvious to me at first just reading its name.

Reply:
This could have also been a `table_view`, but `ping_view` works here too since it has a `client_id` and some `channel`. Using a `ping_view` some additional measures, like a count of clients, are auto generated